### ISSUE TITLE
Problem after updating from `0.1.13` to `0.1.14`. Sveltekit + Typescript + Storybook

### ISSUE THREAD

**Issue by SergeiZheleznov** [eirslett/storybook-builder-vite#224]
Hi! After updating `storybook-builder-vite` version from `0.1.13` to `0.1.14` all stories disappear. 

console:
```Unexpected default export without title in './src/stories/Button.stories.svelte': undefined```

screenshot:
![image](https://user-images.githubusercontent.com/3751791/151874255-f7e218ee-8d44-4b4c-aba9-ae951bbff4c0.png)

Example: https://github.com/SergeiZheleznov/sb-svelte-vite-example

The only thing changed is `storybook-builder-vite` version

**Comment by IanVS** [comment]
Thanks for the report.  I can't think of anything that would have changed to cause this, and unfortunately I'm not able to get our own svelte example working due to https://github.com/eirslett/storybook-builder-vite/issues/50.  Is it possible that @storybook/addon-svelte-csf is not behaving correctly?  I see a warning about `configure()`, where is that coming from?

**Comment by joshwooding** [comment]
The issue is somewhere here: https://github.com/eirslett/storybook-builder-vite/pull/216/files#diff-223dc34a197054cef56af310829bf2af8a89fc72deac2a0faa7ca305b0ed80e1R109 seems like ".svelte" files work differently :?

**Comment by Narigo** [comment]
I can confirm this report with my [SvelteKit boilerplate](https://github.com/Narigo/sveltekit-boilerplate). I have updated all packages today, if you try updating from `storybook-builder-vite@0.1.13` to `storybook-builder-vite@0.1.14`, it will say that it cannot find any stories.

**Comment by thomaswilsonxyz** [comment]
Hi, I've had exactly this problem as well after updating from.  (We're running SvelteKit `1.0.0-next.241`.)

For now, we've solved this by reverting back to `0.1.13`.

Some interesting observations I made while bughunting with a colleague:

- We're using the wildcard `*` to find stories in a relative directory, in which case the frontend is throwing the error `index.js:56 Unable to load story '*':`.  When I changed the `stories` properties in the `main.js` file for storybook to point to one specific story file (e.g. `../stories/Button.stories.svelte`) I get an error pointing towards _another_ story in that directory.   It was the same `Unable to load story...` error, but with a reference to a hashed fildname for another story in that same directory.
- After nuking and re-loading `node_modules`, I got the same `*` error message (same `main.js` file)
-  I (unreliably) got an error telling me that `config.stories` was no longer supported on the storybook `main.js` file.  we're not using the nested `config.stories` structure, but the plain `stories` field, as per the Storybook documentation.  

I'm unsure if there's a reference to another configuration object / file for Storybook, but it's like I couldn't hook into Storybook through the config.
